app/templates/direct-award/end-search.html

```diff
@@ -16,7 +16,7 @@
       },
       {
         "link": url_for('direct_award.saved_search_overview', framework_framework=framework.framework),
-        "label": "Saved Searches"
+        "label": "Your saved Searches"
```

Comment:
'Searches' probably shouldn't be capitalised.

Reply:
ah nice one